library/core/functions.general.php

```diff
@@ -722,6 +722,42 @@ function safePrint($mixed, $returnData = false) {
     }
 }
 
+if (!function_exists('dbdecode')) {
+    /**
+     * Decode a value retrieved from database storage.
+     *
+     * @param string $value
+     * @return mixed
+     */
+    function dbdecode($value) {
+        try {
+            $decodedValue = unserialize($value);
```

Comment:
I think if you put a test for `$value === null` then you are going to catch a very real scenario with null db values and avoid the exception handler.